### ISSUE TITLE
Display frozen at home screen

### ISSUE THREAD

**Issue by lenikhilsingh** [daniel-thompson/wasp-os#358]
Hi,

My Pinetime watch after flashing wasp os is frozen on home screen:

![photo_6118412171827328229_y](https://user-images.githubusercontent.com/30202672/189624512-0c4b0a16-4915-4c01-946d-ce265b59cbeb.jpg)

Now, I want to go back to Infinitime. I'm not able to do it.

The DFU button is not appearing when I try DFU with NRFConnect:

![photo_6116016709652559180_y](https://user-images.githubusercontent.com/30202672/189625135-676c0e58-5ea5-4edb-8ddf-99764e01875e.jpg)

As for Gadetbridge, the device gets disconnected and is not able to connect when I reboot the watch by holding the button until the pine cone arrow appears. I tried another way, by click opening the firmware file with gadgetbridge. Here's the response:

![photo_6118412171827328228_y](https://user-images.githubusercontent.com/30202672/189625809-4291fe4f-dbfe-4803-93f7-fd58a0656116.jpg)

I tried DaFlasher as well. DaFlasher's response:

![photo_6122811390864502976_y](https://user-images.githubusercontent.com/30202672/189631427-d680c3b0-97e9-4c24-85c9-aa6ecd7b093a.jpg)

Please suggest ways to rectify this. So that I can reinstall Infinitime.

**Comment by thiswillbeyourgithub** [comment]
What version of nRF connect are you using?

Maybe take a look at this https://github.com/daniel-thompson/wasp-os/issues/335#issuecomment-1170212449

**Comment by lenikhilsingh** [comment]
Hi, thank you so much for replying.

I'm using [nRF Connect 4.24.3](https://github.com/NordicSemiconductor/Android-nRF-Connect/releases/tag/v4.24.3)

**Comment by lenikhilsingh** [comment]
I'm not able to use dfu, because DFU button doesn't appear in my case. As opposed to https://github.com/daniel-thompson/wasp-os/issues/335#issuecomment-1170212449

**Comment by thiswillbeyourgithub** [comment]
What is the name of the pinetime device you're trying to connect to, as seen by nRF connect ?

Is it something like "pinetime" or "pineDFU" ?

**Comment by lenikhilsingh** [comment]
Name is PINETIME. Anyway, I was wondering, why does it show CONNECTED NOT BONDED. I might not have noticed earlier.

**Comment by thiswillbeyourgithub** [comment]
You have to press the button until the watch reboots into the (IIRC) bootloader mode. In bootloader mode, pressing again the button will boot wasp os.

The DFU button will only appear in bootloader mode.

**Comment by lenikhilsingh** [comment]
yes, I did reboot by hard pressing the power button for sometime. It was on pinetree logo(and start arrow)

The device got disconnected from NRF connect. And as long as it remains on the pinetree logo, I'm not able to connect with the device on nrf Connect. When I press the power button to come back to home screen, then the device gets connected to nrf connect.
But then, after connecting, I'm not able to access DFU button.

![photo_6125063190678188276_y](https://user-images.githubusercontent.com/30202672/189660057-934d3333-da44-4a08-b0bf-19b7c5f2058f.jpg)


![photo_6125063190678188277_y](https://user-images.githubusercontent.com/30202672/189660159-e1350549-2cc8-4dae-8d13-8c298731e6d1.jpg)

**Comment by lenikhilsingh** [comment]
Is there any simpler process to reset the watch and go back to the initial OS, that it came installed with?

**Comment by thiswillbeyourgithub** [comment]
I think you're misuing nRF connect. When switching from "regular mode" to "bootloader" the device changes name from "pinetime" to "pineDFU".

Your picture is showing you're on the wrong device in nRF connect

**Comment by lenikhilsingh** [comment]
It's the same device. Please check the device ID, F4:27:4C:72:75:F8. When I switch from regular to bootloader mode I'm not able to connect. Anyway, what do you think might have gone wrong, when you say I'm misusing the app?

**Comment by thiswillbeyourgithub** [comment]
the tab name in nRF connect is "pinetime" and I think it sould be "pineDFU". Search again for the device once you're in bootloader mode.

**Comment by lenikhilsingh** [comment]
Actually, This is an important piece of information that I missed. I wasn't expecting the name to change to PineDFU, that's why I wasn't scanning for the device again. So, When I scanned again, the device  name "PineDFU" appeared and I flashed the reloader zip: reloader-infinitime-recovery-0.14.1.zip.

Anyway, After this when the flashing was complete, the watch has gone in the loop, where in these two screens appear one after another.

![photo_6125063190678188287_y](https://user-images.githubusercontent.com/30202672/189666600-0702b987-4407-4532-a26b-f60aae77c5a1.jpg)

![photo_6125063190678188288_y](https://user-images.githubusercontent.com/30202672/189666724-0ef0d066-5699-4af3-a244-86b9e49b6395.jpg)


I'm checking @daniel-thompson's comment https://github.com/daniel-thompson/wasp-os/issues/335#issuecomment-1170212449, maybe I can relate with that issue.

**Comment by lenikhilsingh** [comment]
Okay, I followed these steps And flashed pinetime-mcuboot-app-dfu-1.9.0.zip

![Screenshot 2022-09-12 at 7 12 07 PM](https://user-images.githubusercontent.com/30202672/189669568-ffb928e3-3e85-4899-bb47-0aafd16138a8.png)

Now, the device is frozen at Infinitime's home screen:

![photo_6125063190678188292_y](https://user-images.githubusercontent.com/30202672/189669980-53be0056-7722-4dd5-9d49-74894be38c31.jpg)

**Comment by lenikhilsingh** [comment]
Thank you. We cna close this issue now.
